### ISSUE TITLE
Too small loss on first epoch

### ISSUE THREAD

**Issue by mlcoop** [Vipermdl/OCR_detection_IC15#8]
I have tried to train on icdar 2015 dataset.
But loss is already 0.06 after firt epoch. Seems not all right @Vipermdl 
Could you please help me out?

**Comment by novioleo** [comment]
> I have tried to train on icdar 2015 dataset.
> But loss is already 0.06 after firt epoch. Seems not all right @Vipermdl
> Could you please help me out?

not enough,i can reach 0.0028

**Comment by mlcoop** [comment]
@novioleo could you please share your training config like?
-resnet50 or resnet34?
-batch size
-how many epochs did you train?
-didnt you overfit?
-which dataset are you using to train?

**Comment by novioleo** [comment]
1. not overfit at all.the val loss similar to train loss
2. bs=16
3. resnet50
4. icdar2015
@AssylCooper

**Comment by mlcoop** [comment]
@novioleo Thanx
did it take you like weeks to get loss: 0.0028 ?
because it trains really slowly

**Comment by novioleo** [comment]
@AssylCooper change your learning rate~i don't think it slow.i spend few days to train.

**Comment by mlcoop** [comment]
@novioleo I'm trained it on icdar2017(7200 images) and now fine-tuning on icdar2015.
Could you please share your weights with 0.0028 loss? 
Thanx

**Comment by novioleo** [comment]
sorry ,i can't
@AssylCooper
